ServerManager/classes/GameSession.php

```diff
@@ -255,6 +256,7 @@ public function recreate(): bool
         // after a server upgrade the version might have changed since session was first created
         $this->server_version = ServerManager::getInstance()->getCurrentVersion();
         $this->sendCreateRequest(1);
+        // alternative to the above is in editGameSession.php
```

Comment:
left-over that can be removed ?
```suggestion
        // alternative to the above is in editGameSession.php
```